Packs/Zerohack_XDR/Integrations/ZerohackXDR/README.md

```diff
@@ -0,0 +1,2 @@
+Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
```

Comment:
I don't see the beta flag in your yml file. If you want to release this pack as beta add beta:true to the yml, if not remove this comment